Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_faces.h

```diff
@@ -29,6 +29,10 @@
 #include <CGAL/use.h>
 #endif
 
+#include <CGAL/assertions.h>
```

Comment:
AFAIS, all changes in this file can be undo.

Reply:
Probably yes. I will see if this is possible.

Reply:
Done.